src/record/record.js

```diff
@@ -453,9 +459,11 @@ Record.prototype._onRead = function( message ) {
 Record.prototype._setReady = function() {
 	this.isReady = true;
 	for( var i = 0; i < this._queuedMethodCalls.length; i++ ) {
+		// TODO: What if callback throws?
 		this[ this._queuedMethodCalls[ i ].method ].apply( this, this._queuedMethodCalls[ i ].args );
```

Comment:
This will never throw if we properly validate all the arguments in `set` before enqueueing it.